docs/integrations/android-specific-workflow-steps/increment-build-and-version-number.md

```diff
@@ -57,9 +57,9 @@ The outputs that can result from the operation of this component are listed as f
 
 | Variable Name                  | Description                                              |
 |--------------------------------|----------------------------------------------------------|
-| `$AC_ANDROID_NEW_VERSION_CODE` | Incremented version code that is written to the project. |
-| `$AC_ANDROID_NEW_VERSION_NAME` | Incremented version name that is written to the project. |
+| `$AC_ANDROID_NEW_VERSION_CODE` | Incremented version code is written to the project. |
+| `$AC_ANDROID_NEW_VERSION_NAME` | Incremented version name is written to the project. |
```

Comment:
@boztopuz Due to this change, the sentence has turned into a **sentence expressing a situation**, but I think that the 2nd column should be the **definition sentence** of the 1st column.